docs/guides/using-with-vuex.md

```diff
@@ -241,10 +239,11 @@ describe('MyComponent.vue', () => {
 
     store = new Vuex.Store({
       modules: {
-        mymodule: {
+        myModule: {
+          namespaced: true, // If store/myModule is namespaced https://vuex.vuejs.org/guide/modules.html#namespacing
```

Comment:
I don't think this is necessary. I think the examples should have as little noise as possible. Since the user is testing their modules, we can assume they know if they namespaced it or not. Since the focus of this guide and repo is the test setup/assertions, not Vuex and the Vuex API, I don't think this should be included.